class/defaults.yml

```diff
@@ -335,6 +348,13 @@ parameters:
               secretNamespace: ${appcat:services:generic:objectstorage:secretNamespace}
               providerSecretNamespace: ${appcat:providers:cloudscale:connectionSecretNamespace}
               bucketDeletionPolicy: ${appcat:services:generic:objectstorage:bucketDeletionPolicy}
+            minio:
+              enabled: false
+              secretNamespace: ${appcat:services:generic:objectstorage:secretNamespace}
+              providerSecretNamespace: ${appcat:providers:cloudscale:connectionSecretNamespace}
+              bucketDeletionPolicy: ${appcat:services:generic:objectstorage:bucketDeletionPolicy}
+              grpcEndpoint: ${appcat:grpcEndpoint}
```

Comment:
Do we really need the GRPC endpoint to be exposed?

Reply:
I usually set it in `vshn.yaml` when debugging, then I don't have to set it every time after I re-generate.